src/libs/AttachmentUtils.js

```diff
@@ -0,0 +1,27 @@
+import lodashGet from 'lodash/get';
+import _ from 'underscore';
+import * as FileUtils from './fileDownload/FileUtils';
+import CONST from '../CONST';
+import Receipt from './actions/Receipt';
+
+const isValidFile = (translate, _file) => {
+    const {fileExtension} = FileUtils.splitExtensionFromFileName(lodashGet(_file, 'name', ''));
+    if (_.contains(CONST.API_ATTACHMENT_VALIDATIONS.UNALLOWED_EXTENSIONS, fileExtension.toLowerCase())) {
+        Receipt.onUploadReceiptError(true, translate('attachmentPicker.wrongFileType'), translate('attachmentPicker.notAllowedExtension'));
```

Comment:
`AttachmentUtils` should  have nothing to do with `Receipt`. The `Receipt` logic should be moved to where the `isValidFile` function is being called.

Reply:
Renamed to `ReceiptUtils.isValidReceipt`